services/tracing/tracing/transport/mq.go

```diff
@@ -0,0 +1,50 @@
+package tracingtransport
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/rabbitmq/amqp091-go"
+	"sensorbucket.nl/sensorbucket/pkg/mq"
+	"sensorbucket.nl/sensorbucket/services/tracing/tracing"
+)
+
+func StartMQ(svc *tracing.Service, conn *mq.AMQPConnection, errQueue string, queue string) func() {
+	done := make(chan struct{})
+	pipelineMessages := mq.Consume(conn, queue, setupFunc(queue))
+	pipelineErrors := mq.Consume(conn, errQueue, setupFunc(errQueue))
+
+	log.Println("Measurement MQ Transport running")
+	go process(queue, pipelineMessages, svc.HandlePipelineMessage)
+	go process(errQueue, pipelineErrors, svc.HandlePipelineError)
+
+	return func() {
+		close(done)
+	}
+}
```

Comment:
Where is done channel used?

Reply:
nowhere, some copy/paste leftovers. removed it